applications/nrf_desktop/configuration/common/gpio_pins.h

```diff
@@ -3,12 +3,12 @@
  *
  * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
  */
-#ifndef _BUTTONS_H_
-#define _BUTTONS_H_
+#ifndef _GPIO_PINS_H_
```

Comment:
let's extract this change (rename of this structure and affected old files) to the separate commit (and PR since it would be easier to review and merge)

Reply:
btw - it seems we use `static const char * const port_map[]` in several places
maybe it would be a right choice to put this map here since port in the structure means entry in this table

Reply:
Then we would assume that we always use all the ports (currently e.g. for pca10059 there is only one port - port1)

Reply:
yes, we should have all ports here